client/components/layout/Block/__tests__/Block-spec.js

```diff
@@ -1,3 +1,4 @@
+import '__tests__/specHelper'
```

Comment:
Ouch... we have to do this in every test now? Wasn't there an option in mocha opts for something like this?